poster/index.html

```diff
@@ -5,11 +5,37 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles/styles.css">
     <title>Document</title>
 </head>
 
 <body>
-    <h1>Poster</h1>
+    <h1 class="header">Walt Disney's</h1>
+
+    <h2 class="header header--secondary">The Jungle Book</h2>
+
+    <div class="info">
+        <div class="info info--1">Featuring the voices of</div>
```

Comment:
On the `div`s on lines 18-23, I I think you actually want to delete/remove the `info` class on these elements -- that's repeating the grid on each individual cell, which is not what you intend. Luckily, the interior `div`s will inherit all the font rules from the `info` class on the parent `div` on line 17 👍